CHANGELOG.md

```diff
@@ -2,87 +2,63 @@
 
 All notable changes to this project will be documented in this file.
 
-<!--attr-start-->
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-
-<!--attr-end-->
-
-<!--
-Using the following categories, list your changes in this order:
+## [Unreleased]
 
 ### Added
--   for new features.
-
-### Changed
--   for changes in existing functionality.
-
-### Deprecated
--   for soon-to-be removed features.
-
-### Removed
--   for removed features.
-
-### Fixed
--   for bug fixes.
-
-### Security
--   for vulnerability fixes.
- -->
-
-<!--changelog-start-->
-
-## [Unreleased](https://github.com/Archmonger/ServeStatic/compare/2.0.1...HEAD)
 
--   Nothing (yet)!
+- New CI workflow, including utilizing PyPI trusted publishing for releases and creation of GitHub releases
+- Formatting and linting is now provided by `ruff`
 
-## [2.0.1](https://github.com/Archmonger/ServeStatic/compare/2.0.0...2.0.1) - 2024-09-13
+## [2.0.1] - 2024-09-13
 
 ### Fixed
 
--   Fix crash when running `manage.py collectstatic` when Django's `settings.py:STATIC_ROOT` is a `Path` object.
+- Fix crash when running `manage.py collectstatic` when Django's `settings.py:STATIC_ROOT` is a `Path` object.
 
-## [2.0.0](https://github.com/Archmonger/ServeStatic/compare/1.2.0...2.0.0) - 2024-09-12
+## [2.0.0] - 2024-09-12
 
 ### Added
 
--   Django `settings.py:SERVESTATIC_USE_MANIFEST` will allow ServeStatic to use the Django manifest rather than scanning the filesystem.
-    -   When also using ServeStatic's `CompressedManifestStaticFilesStorage` backend, ServeStatic will no longer need to call `os.stat`.
+- Django `settings.py:SERVESTATIC_USE_MANIFEST` will allow ServeStatic to use the Django manifest rather than scanning the filesystem.
```

Comment:
Are these whitespace changes really needed?

Reply:
Reduced the diff here